emperor/support_files/js/controller.js

```diff
@@ -431,14 +445,76 @@ define([
    */
   EmperorController.prototype.screenshot = function(type) {
     type = type || 'png';
-    // Render all scenes so it's rendered in same context as save
-    for (var i = 0; i < this.sceneViews.length; i++) {
-      this.sceneViews[i].render();
+
+    if (type === 'png') {
+      // Render all scenes so it's rendered in same context as save
+      for (var i = 0; i < this.sceneViews.length; i++) {
+        this.sceneViews[i].render();
+      }
+      var c = this.renderer.domElement.toDataURL('image/' + type);
+      // Create DOM-less download link and click it to start download
+      var download = $('<a href="' + c + '" download="emperor.' + type + '">');
+      download.get(0).click();
+    } else if (type === 'svg') {
```

Comment:
Now that we have the option to save as SVG, can you document this in the docstring for this method?